core/src/main/java/org/mule/runtime/core/DefaultAttributes.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core;
+
+import org.mule.runtime.api.message.Attributes;
+
+/**
+ * Default implementation of {@link Attributes} to be used when no other connector specific attributes instance is set.
+ */
+public class DefaultAttributes implements Attributes
```

Comment:
I don't see the value of this class. Besides if this exists, then why do you use `new Attributes{}` above?